src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.java

```diff
@@ -182,17 +201,6 @@ public UserProfileEditorWidgetViewImpl(
     locationTextBox = locationSuggestBox.getTextBox();
     locationTextBox.addStyleName("form-control");
     locationSuggestBoxContainer.add(locationSuggestBox);
-    // note, not adding email since it's not editable here.
-    textBoxes =
```

Comment:
Previously, this code would run through this array of text boxes and set them to readOnly if we were not editing.  Now, there are "renderer" (static, read-only) elements that are shown instead.